src/en/models-cmr-scene-2.md

```diff
@@ -244,104 +257,32 @@ ian:admin/default.hosted-mysql:
       access: read
 ```
 
-To find offers in a specified model:
+To find offers in model `cmr-model-1` on controller `lxd-cmr-1`:
 
 ```bash
-juju find-offers admin/default
-juju find-offers ian:admin/default
+juju find-offers lxd-cmr-1:admin/cmr-model-1
```

Comment:
is it always necessary, when targeting a model, to include the user name too? I mean, it seems to be from my experiments, but is that really true?

I guess what I am saying is that I would assume 

juju find-offers lxd-cmr-1:cmr-model-1 

would work. It doesn't. Am I missing something?

Reply:
The username pertains to the offering side and I don't see how the consuming side can guess what it is.

Reply:
Sorry, I misunderstood.
So what you say makes sense and I tried it and it works for me. :rabbit: 
```
juju find-offers lxd-cmr-1:cmr-model-1
Store      URL                      Access  Interfaces
lxd-cmr-1  admin/cmr-model-1.mysql  admin   mysql:db

Reply:
That is even more worrying! I will try again tomorrow